test/docker/compose.go

```diff
@@ -575,9 +574,9 @@ func (d *Compose) startCluster(ctx context.Context, size int, settings map[strin
 		config3["CLUSTER_DATA_BIND_PORT"] = "7105"
 		config3["CLUSTER_JOIN"] = fmt.Sprintf("%s:7100", Weaviate1)
 		eg.Go(func() (err error) {
-			time.Sleep(time.Second * 3)
```

Comment:
Due to the peculiarities of member-list bootstrapping in a cluster, t his delay may be necessary to prevent acceptance test failures